### DISCUSSION TITLE
400 Error Connection Issue - Post Products in Sellbrite

### DISCUSSION THREAD

**Discussion by oarnb** [Sellbrite/Sellbrite-API#17] [Q&A]
Please help me as I'm having 400 Error with my connection in posting products on Sellbrite. 

See attached

![Screenshot_7](https://github.com/Sellbrite/Sellbrite-API/assets/138830828/661c9a8d-f2a9-4dd2-92d5-6652de9cddf8)

**Comment by rengler-godaddy** [comment]
Hello - Thanks for reaching out! 

Are you able to post the full request with the body params? That might help us see what’s causing the 400 error.
